css/style.css

```diff
@@ -491,7 +491,8 @@ a:hover {
     }
 
     .row {
-        padding: 2em;
+        padding-left: 4em 6.6em;
+        padding-right: 4em 6.6em;
```

Comment:
The CSS `padding` property can be set to 1-4 numbers but the individual padding properties (`padding-left`, `padding-right` and such) should only have one value.

See: https://www.w3.org/TR/2011/REC-CSS2-20110607/box.html#propdef-padding-left

Reply:
My bad! Thanks for catching that @simonbates. I'll send a PR to address that.